README.md

```diff
@@ -18,33 +18,51 @@ var geoDb = require('fxa-geodb')({
 
 --
 ### API
-The function returns a promise that may either resolve (on successful finding of location data) or reject (if either the ip was invalid, or location data could not be found). Call the function, like so:
+The function returns a location object
+or throws if the ip was invalid
+or location data could not be found.
+Call the function like so:
 
 ```JavaScript
-geoDb(ip)
-  .then(function (location) {
-    // success, resolved
-    // location data is available here
-  }, function (err) {
-    // rejected :(
-    // Uh-oh error
-  });
+try {
+  const location = geoDb(ip);
+  // Use location...
+} catch (err) {
+  // Handle err
+}
```

Comment:
we should provide an example of how to wrap the API into a promise....



.... just kidding 😆